@types/es-module-lexer/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module "es-module-lexer" {
+  export type ImportSpecifier = {
+    s: number,
+    e: number,
+    ss: number,
+    se: number,
+    d: number
+  };
+  export function init(): Promise<void>;
+  export function parse(code: string): [Array<ImportSpecifier>, Array<string>];
+}
```

Comment:
Nice! Maybe if you have time you could contribute these to https://github.com/DefinitelyTyped/DefinitelyTyped, then we can get help from others maintaining this (unless the package author would like to ship these types themselves)

Reply:
Totally, that was my plan after I'd got the types nailed for our use case, will definitely be contributing them to DefinitelyTyped